data/json/items/comestibles.json

```diff
@@ -11622,6 +11622,35 @@
     },
     {
         "type" : "COMESTIBLE",
+        "id" : "flatbread",
+        "name" : "flatbread",
+        "weight" : 56,
+        "color" : "brown",
+        "addiction_type" : "none",
+        "spoils_in" : 280,
+        "stim" : 0,
+        "container" : "bag_plastic",
+        "to_hit" : 0,
+        "comestible_type" : "FOOD",
+        "symbol" : "%",
+        "quench" : 0,
+        "healthy" : 1,
+        "addiction_potential" : 0,
+        "nutrition" : 15,
+        "description" : "Simple unleavened bread.",
+        "price" : 140,
+        "material" : "wheat",
+        "tool" : "null",
+        "volume" : 2,
+        "cutting" : 0,
+        "phase" : "solid",
+        "charges" : 8,
+        "bashing" : 0,
+        "flags" : ["EATEN_HOT"],
+        "fun" : 1
```

Comment:
I can't really think of food blander than fried flour... 0 would fit better. Also that "EATEN_HOT" doesn't fit.